### PR TITLE
Ship queryTemplate to v1.0 doc

### PR DESCRIPTION
**Pull request by yiwenwang-ms**

### CODE REVIEW

**Comment by FaithOmbongi** [comment]
Hi @angelgolfer-ms, @Jarbas-MSFT, @Danielabom - FYI

**Comment by Danielabom** [comment]
Contacted author to get info.

File: api-reference/v1.0/api/search-query.md [hunk 1]
```diff
@@ -52,104 +52,136 @@ If successful, this method returns `HTTP 200 OK` response code and a [searchResp
 
 ## Examples
 
-### Request
+### Example 1: Basic call to perform a search request
 
-The following is an example of the request.
+The following example shows how to perform a search to get expected connector items.
```
Comment (Danielabom) [inline api-reference/v1.0/api/search-query.md:57]:
```suggestion
The following example shows how to search for expected connector items.
```

File: api-reference/v1.0/api/search-query.md [hunk 2]
```diff
@@ -52,104 +52,136 @@ If successful, this method returns `HTTP 200 OK` response code and a [searchResp
...
+### Example 1: Basic call to perform a search request
 
-The following is an example of the request.
+The following example shows how to search for expected connector items.
+
+#### Request
 
 ```HTTP
-POST https://graph.microsoft.com/v1.0/search/query
+POST https://graph.microsoft.com/beta/search/query
```
Comment (Danielabom) [inline api-reference/v1.0/api/search-query.md:62]: Quick question, why did you change the version?  Thanks.

Comment (yiwenwang-ms) [reply api-reference/v1.0/api/search-query.md]: Thank you, that's a mistake. Corrected.

File: changelog/Microsoft.Search.json [hunk 3]
```diff
@@ -1599,6 +1599,24 @@
...
       "SubArea": ""
+    },
+    {
+      "ChangeList": [
+        {
+          "Id": "affa85e4-b3d9-4c62-a8a9-a39c0fb6f866",
+          "ApiChange": "Property",
+          "ChangedApiName": "queryTemplate",
+          "ChangeType": "Addition",
+          "Description": "Added queryTemplate to the search [query](https://learn.microsoft.com/en-us/graph/api/search-query?view=graph-rest-1.0).",
```
Comment (Danielabom) [inline changelog/Microsoft.Search.json:1610]: This is the link for the action and the "queryTemplate" property belongs to the searchQuery resource, please update the link in the description. Thanks.

**Review by Danielabom — CHANGES_REQUESTED** [review]
@yiwenwang-ms, thank you for working on this PR. Please review my comments.

File: changelog/Microsoft.Search.json (hunk 3 above)
Comment (yiwenwang-ms) [reply changelog/Microsoft.Search.json]: Thanks, done!

**Pull request merged**